project.clj

```diff
@@ -141,7 +147,7 @@
                                   (require 'schema.core)
                                   (schema.core/set-fn-validation! true))]}
              :ezbake {:dependencies ^:replace [[puppetlabs/puppetdb ~pdb-version]
-                                               [org.clojure/tools.nrepl "0.2.3"]]
+                                               [org.clojure/tools.nrepl nil]]
```

Comment:
unfortunately we need this, I added the notes/comment from this section of the puppetserver project.clj as well to explain why